### DISCUSSION TITLE
Type error: Route "src/app/api/auth/[...nextauth]/route.ts" does not match the required types of a Next.js Route.   "handler" is not a valid Route export field.

### DISCUSSION THREAD

**Discussion by aypen-git** [nextauthjs/next-auth#9463] [Help]
Code of src/app/api/auth/[...nextauth]/route.ts :

import NextAuth from "next-auth";
import { authOptions } from "../../../../../lib/authUtils";

  export const handler = NextAuth(authOptions);
  export { handler as GET, handler as POST };

**Comment by aypen-git** [comment]
This error is occurring on npm build.

**Reply by blakesekelsky** [reply]
have you ever found a fix? currently dealing with this trying to deploy to railway

**Comment by abdtriedcoding** [comment]
Any updates ??

**Comment by PabloMartinez-Beemeral** [comment]
I removed the "export" from the handler and for now it works fine.
